source/_posts/Rails-Starter-Template.md

```diff
@@ -0,0 +1,209 @@
+---
+title: Rails Starter Template
```

Comment:
use few keyword from this list as we need for increasing the ranking of the page

NAYAN
Traffic monitoring
AI
Livelihood
Driver analytics
Violation mapping
Better commute
Monitor infrastructure defects
Safer roads
Road Safety
Traffic Violation
Artificial Intelligence
India
Smart City
Safety
Data Protection
PARKING DETECTION, 
VEHICLE OCCUPANCY DETECTION
NAYANINDIA
USER INTERACTION
NAYAN IP
NAYAN DATA PROTECTION
Road User
Public Road
Vehicle
Dangers
MACHINE LEARNING

Reply:
Keywords we can use only those that are relevant to the content of the blog. As none of these keywords are relevant here, I have added a line at the bottom of the blog mentioning "Nayan, Traffic monitoring and road safety"